non-normative/recipes.adoc

```diff
@@ -30,3 +30,18 @@ configuration utilities. The actual meaning of the default (compatible to
 BRS-I) configuration would be highly specific to the vendor. In this example,
 it could capping all harts at the same performance level, or it could mean
 disabling efficiency harts.
+
+[[recipe-brs-m-guidance]]
+===== BRS-M Recipe Guidance
+
+In contrast to BRS-I and BRS-B, BRS-M has no implementation of UEFI related
+services. Non-UEFI firmware, such as *coreboot*, can also supports boot well.
```

Comment:
put coreboot in the bibtex and reference it from here?

Reply:
Why do you relate to Coreboot here? Isn't any SBI implementation enough to boot an OS (e.g. Linux).

Reply:
My suggestion was about citing the external references. Since coreboot is newly introduced it should be cited.

Reply:
I mistook xypron's comment as a response to my comment.  That seems to not be the case.